schedule/yam/agama/agama_install_with_cryptfs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: agama_installation
+description: >
+  Playwright test on agamaq
+vars:
+  ENCRYPT: 1
```

Comment:
vars should be in job group yaml, having them in the yaml creates more chances to need more yaml files.